target/linux/airoha/files/drivers/pinctrl/pinctrl-airoha.c

```diff
@@ -623,6 +685,30 @@ static const char *const serdes_groups[] = { "ssusbp0_serdes",
 					     "xpon_serdes",
 					     "ssusbp1_serdes" };
 
+static const char *const phy1_led0_groups[] = { "gpio33", "gpio34",
+					        "gpio35",  "gpio42" };
```

Comment:
@merbanan what is the difference of having a single led group here?

Reply:
Per my understanding of the code macros the groups need to have these names to match the function declaration. The led map register makes it possible to mux the phy[1-4] led0 signals to any of the led0 gpio pins. And the same for the led1 gpio pins. This equals to 8 functions and thus we can not have only one led group.